en/user_guide/global_page_blocks.rst

```diff
@@ -1,23 +1,35 @@
 Global Pages and Blocks
 =======================
 
-In Open Orchestra, one of the purposes is to simplify the creation of pages by using blocks including their own functionnality.
-They must be configured according to this functionnality (see `Block creation`_ and `Block list`_).
+In Open Orchestra, to limit the user work, we provide a way to contribute some part of
+a website once it for all. This usage is grouped under the term of **Global page** and
+**Global blocks**.
 
-Some blocks could have the same configuration parameters in many pages like header or footer block.
-Global pages are containers allowing to store these globally configured blocks.
-Such blocks are reusable in all pages.
+Global pages
+------------
 
-They will appear with a blue background in the backoffice right panel whereas the other blocks appear in orange color.
-
-.. image:: ../../images/panel_global_page_blocks.png
-
-The global pages could be accessed through the left menu of the backoffice.
+Global pages have several languages, don't have a `workflow`_, have only one version and can
```

Comment:
Before describing golbal pages, it could be nice to tell how it mainly works. The general introduction is too vague.